### DISCUSSION TITLE
eta(T) in rheoHeatFoam

### DISCUSSION THREAD

**Discussion by Joe199999** [fppimenta/rheoTool#134] [General]
Dear developers
I want to use the Arrhenius function for the channel case in tutorials for both eta and lambda. I changed the channel case to a scaled problem, i.e. I set Tw=0, Tin=1, Uin=1, H=1, ... and calculated the fluid properties according to non-dimensional groups(Re, Pr, Wi, ...). However, when I run my case, the problem would be diverged! I read something about it in the user guide, but I couldn't solve it!
I used the Arrhenius function individually for eta or lambda one time and for both of them another time. 
When I changed the Tw from 0 to a non-zero value, the problem was solved! But I want to solve a scaled case.

In addition, I wanted to use the modified Arrhenius function, but I didn't know how to calculate alpha in this model from the alpha in the standard Arrhenius model. In simple calculations, alpha in the modified Arrhenius model is a function of T: alpha_m = alpha/(T*Tref)

I appreciate your time to investigate my problem

**Comment by Joe199999** [comment]
Update:
I copied the Arrhenius thermo function in rheoTool libraries and defined a new temperature in it. So, the temperature, which is entered to the Arrhenius model, will be the physical values, not scaled and I got the correct output from the Arrhenius function and I repeated Nobrega's results.

**Reply by milad092** [reply]
Hi Joe!
I hope you are doing well.
I plan to solve these equations in non-dimensional form with rheoheatfoam solver.
Please let me know if you modify the solver or just the constitutive libraries.

**Reply by Joe199999** [reply]
Hi Milad
Sorry for the delay!
No, I didn't make any changes in the solver. I tried to solve the problem in a semi-non-dim mode.
